platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBeanTest.java

```diff
@@ -190,22 +238,45 @@ public void testInit() throws Exception {
     @Test
     public void testDestroy() throws Exception {
         ApplicationServiceBean serviceBean = new ApplicationServiceBean(testAppService,
-                testConfigAdminExt);
-        serviceBean.init();
+                testConfigAdminExt, mBeanServer);
+
+        serviceBean.destroy();
+
+        verify(mBeanServer).unregisterMBean(objectName);
+    }
+
+    /**
+     * Tests the {@link ApplicationServiceBean#destroy()} method
+     * for the case where an InstanceNotFoundException is thrown by mBeanServer.unregisterMBean(...)
+     *
+     * @throws Exception
+     */
+    @Test(expected = ApplicationServiceException.class)
+    public void testDestroyINFException() throws Exception {
```

Comment:
I would name this `testDestroyWhenUnregisterMBeanThrowsInstanceNotFoundException()` or something similar. I know it's a long name but then you don't need to read the comments to know what's being tested.